python/GafferUI/PlugValueWidget.py

```diff
@@ -453,6 +454,17 @@ def __plugFlagsChanged( self, plug ) :
 		if plug.isSame( self.__plug ) :
 			self._updateFromPlug()
 
+	def __plugMetadataChanged( self, nodeTypeId, plugPath, key, plug ) :
+
+		if self.__plug is None :
+			return
+
+		if (
+			Gaffer.affectedByChange( self.__plug, nodeTypeId, plugPath, plug ) or
```

Comment:
I was thinking you would make me a ticket in Shotgun and I would do it then, and was also quietly hoping it might get pushed down the list for a day when I was 100% convinced I really would prefer it that way...